### ISSUE TITLE
Documentation still mentions Eclipse 2019-09 but 2019-12 is already supported

### ISSUE THREAD

**Issue by gregorwolf** [sapmentors/cap-community#55]
Hello SAP CAP Documentation Team,

in the section [Add the SAP Cloud Business Application Tools for Eclipse](https://cap.cloud.sap/docs/java/getting-started#add-the-sap-cloud-business-application-tools-for-eclipse) of the page Java > Getting Started the URL mentioned there is https://tools.hana.ondemand.com/2019-09/. But when I check https://tools.hana.ondemand.com/#cloud then I see also https://tools.hana.ondemand.com/2019-12/. So maybe it's better to add a link to https://tools.hana.ondemand.com/#cloud instead of updating the documentation all 3 month.

Best regards
Gregor

**Comment by iwonahahn** [comment]
Hi Gregor,

Thank you for reporting this. We will update our docs asap in sync with the dev colleagues.

Best regards,
Iwona

**Comment by iwonahahn** [comment]
Hi Gregor,

The colleagues added an [anchor](https://tools.hana.ondemand.com/#cloud-eclipsecds) to the specific section in https://tools.hana.ondemand.com/#cloud. Once the link works I´ll republish capire to reflect the changes and inform here.

Best regards,
Iwona

**Comment by iwonahahn** [comment]
Hi @gregorwolf ,

We have updated capire and removed the specific version by replacing with a link to https://tools.hana.ondemand.com/#cloud-vscodecds, see https://cap.cloud.sap/docs/java/getting-started#add-the-sap-cloud-business-application-tools-for-eclipse.

Best regards,
Iwona

**Comment by gregorwolf** [comment]
I think for the Java tools the link https://tools.hana.ondemand.com/#cloud-eclipsecds would be a better fit.

**Comment by iwonahahn** [comment]
@gregorwolf you are right, my mistake, mixed up the anchor links for the VS Code extension and the tools. Is fixed now and will be republished end of the week. Thanks for the hint :).

**Comment by iwonahahn** [comment]
Capire was republished last Friday, issue can be closed :).
